api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -1379,6 +1379,24 @@ public void changePassword_shouldThrowAPIExceptionIfOldPasswordIsNotCorrect() {
 		assertThat(exception.getMessage(), is(messages.getMessage("error.privilegesRequired", new Object[] {PrivilegeConstants.EDIT_USER_PASSWORDS}, null)));
 	}
 	
+	/**
+	 * @see UserService#changePassword(User,String,String)
+	 */
+	@Test
+	public void changePassword_shouldThrowAPIExceptionIfNewPasswordIsTheSameAsOld() {
+		executeDataSet(XML_FILENAME_WITH_DATA_FOR_CHANGE_PASSWORD_ACTION);
```

Comment:
do you have XML_FILENAME_WITH_DATA_FOR_CHANGE_PASSWORD_ACTION this any where?

Reply:
yeah! it is pointing to https://github.com/openmrs/openmrs-core/blob/master/api/src/test/resources/org/openmrs/api/include/UserServiceTest-changePasswordAction.xml